addons/l10n_in_ewaybill_stock/views/l10n_in_ewaybill_views.xml

```diff
@@ -124,20 +124,7 @@
                         <group>
                         </group>
                     </group>
-                    <notebook>
-                    <page string="Item Details">
-                        <field name="move_ids" mode="tree,kanban" force_save="1">
```

Comment:
Is not it easier for the user to have the information here immediately?  What is the reason you want the stat button instead?

Reply:
@jco-odoo Zoho uses the kind of same functionality for eway-bill they open new view for the lines

cc- @jva-odoo

Reply:
Is not it clearly more user friendly if you see the numbers immediately?

Reply:
@jco-odoo I do agree but then the compute will be an issue until the e-waybill is saved by the user. If you have suggestion for that I might work upto it.

We have added the stat button because user won't be able to click the button unless the Ewaybill is saved.

Reply:
Previously the computation worked because we got the `False` state that is what we are trying to fix it here.

Reply:
Weird, because by default the state should be pending.